lib/ansible/modules/net_tools/netbox/netbox_device.py

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, David Gomez (@amb1s1) <david.gomez@networktocode.com>
+# Copyright: (c) 2018, Mikhail Yohman (@fragmentedpacket) <mikhail.yohman@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: netbox_device
+short_description: Manage devices
```

Comment:
Needs to include the platform. This string is used on https://docs.ansible.com/ansible/latest/modules/list_of_all_modules.html

Reply:
@gundalow We discussed this over slack and it has been updated. Thanks btw!

@amb1s1 also said he wouldn't mind being pinged on any changes so we'll keep him on there.

Is it fine if we merge this now?